swagger_django_generator/templates/django/utils.py

```diff
@@ -57,6 +58,12 @@ def wrapper(request, *args, **kwargs):
                         request.user = user
                         return view_func(request, *args, **kwargs)
 
+        if "HTTP_X_API_KEY" in request.META:
+            key = request.META["HTTP_X_API_KEY"]
+            keys = set(os.getenv("ALLOWED_API_KEYS").split(","))
```

Comment:
Can you make this a global, please: `ALLOWED_API_KEYS = set(os.getenv("ALLOWED_API_KEYS").split(","))`
And then just change the line below to `if key in ALLOWED_API_KEYS:`
The reason for this is that expensive task of retrieving the keys from the environment, splitting it and building a set is only done once (when the module is loaded) and not for every API call.